src/main/java/view/impl/UsersMenu.java

```diff
@@ -0,0 +1,71 @@
+package view.impl;
+
+import dao.UserDao;
+import model.User;
+import service.UserService;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+
+public class UsersMenu {
+    public void runMenu() {
+
+        int input = showMainMenu();
+
+        if (input == 0) {
```

Comment:
a lot of magic numbers

Reply:
fixed. removed magic numbers and created constants